contracts/golos.vesting/config.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+// TODO test params
+#define OUTPUT_WEEK_COUNT 10
+#define OUTPUT_PAYOUT_PERIOD 15
+
+//#define OUTPUT_WEEK_COUNT 13
+//#define OUTPUT_PAYOUT_PERIOD 604800
+
+#define TIMEOUT 3
+#define WEEK_PERIOD 1
+#define NOT_TOKENS_WEEK_PERIOD 0
```

Comment:
Please remove WEEK_PERIOD & NOT_TOKENS_WEEK_PERIOD. The use of these constants is more confusing than useful.